docs/document/content/reference/distsql/syntax/rdl/rule-definition/single-table/create-default-single-table-rule.en.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "CREATE DEFAULT SINGLE TABLE RULE"
+weight = 2
++++
+
+## Description
+
+The `CREATE DEFAULT SINGLE TABLE RULE` syntax is used to create a default single table rule.
+
+### Syntax
+
+```sql
+CreateDefaultSingleTableRule ::=
+  'CREATE' 'DEFAULT' 'SINGLE` `TABLE` `RULE' singleTableDefinition
+
+singleTableDefinition ::=
+  'RESOURCE' '=' resourceName
+
+resourceName ::=
+  identifier
+```
+
+### Supplement
+
+- `RESOURCE` needs to use data source resource managed by RDL
```

Comment:
There is a full stop here in the Chinese page, should the English page be the same?